docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -546,6 +547,16 @@ To use custom values in an Ingress rule define these annotation:
 nginx.ingress.kubernetes.io/proxy-buffering: "on"
 ```
 
+### Proxy buffers Number
+
+Sets the number of the buffers  in [`proxy_buffers`](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers) used for reading the first part of the response received from the proxied server.
```

Comment:
This is not just about reading the first part, what you're describing `proxy_buffer_size`